.github/workflows/create-project.yml

```diff
@@ -26,4 +26,4 @@ jobs:
 
       - name: Create Drupal Project
         run: |
-          composer create-project php-tuf/drupal-project drupal-project-local --repository '{"type": "path", "url": "drupal-project-checkout", "options": {"symlink": false}}' --stability=dev
+          composer create-project -vvv php-tuf/drupal-project drupal-project-local --repository '{"type": "path", "url": "drupal-project-checkout", "options": {"symlink": false}}' --stability=dev
```

Comment:
I think since this project solely used core reviewers and others wanting to review these related packages, I think this is useful to have. It shows that `composer-integration` plugin is actually validating TUF